10.gene-expression-signatures/README.md

```diff
@@ -0,0 +1,20 @@
+# Detecting Gene Expression Signatures
+
+**Gregory Way 2019**
+
+In this module we perform a series of t-tests to detect sample sex in GTEx samples and TCGA patients as well as MYCN amplification status in neuroblastoma (NBL) tumors using TARGET data.
```

Comment:
"detect sample sex based on gene expression profiles ..."  Maybe make the MYCN amplification its own sentence.  I had to read it a couple of times to understand what you were comparing.

Reply:
:+1: